test/controllers/observer_controller_test.rb

```diff
@@ -3774,7 +3774,7 @@ def test_print_labels
     assert_operator(query.num_results, :>=, 4)
     get(:print_labels, params: { q: query.id.alphabetize })
     assert_select("div#labels td", query.num_results)
-    assert_match(/314159/, @response.body) # make sure mycoflora id in there!
+    assert_match(/314159/, @response.body) # make sure fundis id in there!
```

Comment:
Not a blocker, but can some operation be substituted for the magic number "314159"?